### PR TITLE
istio hack script: cluster name should not be set to empty string

### PR DESCRIPTION
**Pull request by josunect**
Fixes https://github.com/kiali/kiali/issues/6751

### CODE REVIEW

File: hack/istio/install-istio-via-istioctl.sh [hunk 1]
```diff
@@ -425,16 +425,21 @@ if [ "${REDUCE_RESOURCES}" == "true" ]; then
     --set components.pilot.k8s.resources.requests.memory=1Mi"
 fi
 
+if [ "${CLUSTER_NAME}" != "" ]; then
+     CLUSTER_NAME_OPTION=" \
+       --set values.global.multiCluster.clusterName=${CLUSTER_NAME}"
+   fi
```
Comment (jmazzitelli) [inline hack/istio/install-istio-via-istioctl.sh:431]: Just some reformatting.. The "fi" is indented incorrectly and no need for a new line in the cluster name option value.
```suggestion
if [ "${CLUSTER_NAME}" != "" ]; then
  CLUSTER_NAME_OPTION="--set values.global.multiCluster.clusterName=${CLUSTER_NAME}"
fi
```

**Review by jmazzitelli — CHANGES_REQUESTED** [review]
some formatting things, but the code looks ok

**Comment by jmazzitelli** [comment]
I took a look at when we changed this (in the past, we always had a cluster name, I wondered when it changed to default to empty string)... the change was here: https://github.com/kiali/kiali/commit/5733e3e675d9e71c68821e209dece7815257d822

@josunect I think you should do the same fix you did here for the other two that are now defaulting to empty string - specifically NETWORK and MESH_ID. Both used to always be set, but now are empty strings.

**Comment by josunect** [comment]
Thanks for pointing at this, @jmazzitelli 
I've applied the same to MESH_ID and NETWORK. 

I was doing some test and doesn't look like there is an additional cluster indicators for local cluster: 

image

I still have the problem not setting the cluster_name in the config. This seems to be fixed by: https://github.com/kiali/kiali/pull/6746

So, not sure if this change have a lot of impact. Said that, I guess it would prevent to set an empty string instead of using the defaults.

**Review by jmazzitelli — APPROVED** [review]
I like this PR better than how it is working today. Rather than explicitly setting things to empty string, I would rather we not set anything and fallback to default Istio behavior.

So I'm +1 on this. approving.

**Review by jshaughn — APPROVED** [review]
+1 to this change, not explcitly setting to `""` makes sense, even if it doesn't change the behavior.

**Pull request merged**